css/main.css

```diff
@@ -325,3 +325,120 @@ li {
 .skills {
   list-style: inside url("../media/svg/circle.svg");
 }
+
+footer {
+  margin: 120px 0 0 0;
+}
+
+.contact {
+  padding: 0 24px;
+}
+
+.description-contact {
+  font-family: Inter, sans-serif;
+  font-size: 32px;
+  font-weight: 700;
+  line-height: 44px;
+  letter-spacing: 0;
+  text-align: center;
+  margin: 0 0 73px 0;
+  color: #172b4d;
+}
+
+.contact-inputs {
+  list-style: none;
+  display: flex;
+  flex-direction: column;
+  align-items: stretch;
+}
+
+#full-name {
+  border-top: none;
+  border-right: none;
+  border-left: none;
+  border-width: 1px;
+  width: 100%;
+}
+
+#email-address {
+  border-top: none;
+  border-right: none;
+  border-left: none;
+  border-width: 1px;
+  margin-top: 39px;
+  width: 100%;
+  margin-bottom: 10px;
+}
+
+.contact-inputs .small-validation {
+  background-color: #172b4d;
+  color: white;
+  padding: 5px 10px;
+}
+
+#text-area {
+  border: none;
+  background-color: #fbf8f7;
+  margin-top: 24px;
+  width: 80%;
+  height: 114px;
+}
+
+.container-button {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  padding: 24px 0 124px 0;
+}
+
+.button-submit {
+  font-family: Inter, sans-serif;
+  font-weight: 600;
+  height: 48px;
+  width: 131px;
+  background-color: #ff6b00;
+  color: white;
+  border: none;
+}
+
+.button-submit:active {
+  background-color: #e05304;
+  color: #e5e5e5;
+}
+
+.button-submit:disabled {
+  background-color: #f4f1f0;
+  color: #5b5957;
+}
```

Comment:
Excellent job working on the styling of the form! 👍🏼  But, you need to implement the style for the [`:hover` pseudoclass](https://developer.mozilla.org/en-US/docs/Web/CSS/:hover) for the submit button.

The reason to implement this is that it improves the UX in the desktop version of your app.
Be careful to follow the exact hover style of the Figma template. You can do this! 😄
![image](https://user-images.githubusercontent.com/97407840/197101164-3ac30138-3a4d-4c12-a03d-526e45b2e911.png)


Tip: You can see your `:hover` property in action on your browser developer tools.
![image](https://user-images.githubusercontent.com/97407840/197100927-e60aa788-d630-4431-9b58-6ae321941762.png)